test/functional/wallet_labels.py

```diff
@@ -33,8 +33,8 @@ class WalletLabelsTest(UnitETestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
-        self.extra_args = [[]] * 4
-        self.extra_args[0] = ['-deprecatedrpc=accounts']
+        self.extra_args = [['-mocktime=1500000000']] * 4
+        self.extra_args[0] += ['-deprecatedrpc=accounts']
```

Comment:
nit: When appending one single element, it's better to rely on `append` than on `extend`:
```python
self.extra_args[0].append('-deprecatedrpc=accounts')
```
Although it's also OK as it is now.